rust/agama-cli/src/main.rs

```diff
@@ -119,6 +132,8 @@ async fn build_manager<'a>() -> anyhow::Result<ManagerClient<'a>> {
 }
 
 async fn run_command(cli: Cli) -> Result<(), ServiceError> {
+    let client = BaseHTTPClient::new()?;
```

Comment:
as mention above, BaseHTTPClient should be used only for commands that already require authenticated access. Or you need to modify it at https://github.com/openSUSE/agama/blob/895b81955c3ed882015ffc7fd4a683a09b4d5762/rust/agama-lib/src/base_http_client.rs#L31 as comment suggest to have constructor that does not set authentication token.
But then you should create that client only in calls where you know that authorization is not required.

Reply:
already done.  Sadly I pushed incomplete stuff before going to vacation :-/